lib/plugins/cors.js

```diff
@@ -80,15 +89,22 @@ function matchOrigin(req, origins) {
 function cors(opts) {
     assert.optionalObject(opts, 'options');
     opts = opts || {};
-    assert.optionalArrayOfString(opts.origins, 'options.origins');
     assert.optionalBool(opts.credentials, 'options.credentials');
     assert.optionalArrayOfString(opts.headers, 'options.headers');
 
+    if (opts.origins) {
+        assert.ok(Array.isArray(opts.origins), 'options.credentials');
```

Comment:
That was a copy-paste error - thanks! :)
As for `assert.array`, I would use it but the problem is that the version of `assert-plus` in use (0.1.5) does not have that method available. I didn't want to mess with the dependencies in this PR, so I had to fall back to a more generic `assert.ok`. If you're ok with that, I can update `assert-plus` to the latest version (0.2.0), so that we can use the new method. I have checked it locally, and updating breaks one test, though it looks like it would be easy to fix.